tests/trainers/test_detection.py

```diff
@@ -17,7 +17,17 @@ class TestObjectDetectionTask:
     @pytest.mark.parametrize(
         "name,classname", [("nasa_marine_debris", NASAMarineDebrisDataModule)]
     )
-    def test_trainer(self, name: str, classname: Type[LightningDataModule]) -> None:
+    @pytest.mark.parametrize(
+        "model_name,backbone",
+        [("faster-rcnn", "resnet18"), ("fcos", "resnet18"), ("retinanet", "resnet18")],
```

Comment:
If we're always testing resnet18, there's no need to parametrize it, we can just use the default in `model_kwargs`. If we had more detection datamodules, I would probably just swap out the module in the config file itself instead of parametrizing, but we can change that in the future.